components/tag/index.css

```diff
@@ -306,106 +298,73 @@
-		color: var(--highcontrast-tag-content-color-emphasized, var(--mod-tag-content-color-emphasized, var(--spectrum-tag-content-color-emphasized)));
+	/* mod/highcontrast colors specified at top */
+	border-color: var(--spectrum-tag-border-color-emphasized);
+	background-color: var(--spectrum-tag-background-color-emphasized);
+	color: var(--spectrum-tag-content-color-emphasized);
+
+	&:not(.is-disabled, .is-readOnly):hover {
+		border-color: var(--spectrum-tag-border-color-emphasized);
+		background-color: var(--spectrum-tag-background-color-emphasized-hover);
+		color: var(--spectrum-tag-content-color-emphasized);
 	}
 
-	&:active {
-		border-color: var(--highcontrast-tag-border-color-emphasized-active, var(--mod-tag-border-color-emphasized-active, var(--spectrum-tag-border-color-emphasized-active)));
-		background-color: var(--highcontrast-tag-background-color-emphasized-active, var(--mod-tag-background-color-emphasized-active, var(--spectrum-tag-background-color-emphasized-active)));
+	&:not(.is-disabled, .is-readOnly):active {
+		border-color: var(--spectrum-tag-border-color-emphasized);
+		background-color: var(--spectrum-tag-background-color-emphasized-active);
+		color: var(--spectrum-tag-content-color-emphasized);
 	}
 
-	&:focus-visible,
-	&.is-focused {
-		border-color: var(--highcontrast-tag-border-color-emphasized-focus, var(--mod-tag-border-color-emphasized-focus, var(--spectrum-tag-border-color-emphasized-focus)));
-		background-color: var(--highcontrast-tag-background-color-emphasized-focus, var(--mod-tag-background-color-emphasized-focus, var(--spectrum-tag-background-color-emphasized-focus)));
+	&:not(.is-disabled, .is-readOnly):focus-visible,
+	&:not(.is-disabled, .is-readOnly).is-focused {
+		border-color: var(--spectrum-tag-border-color-emphasized);
+		background-color: var(--spectrum-tag-background-color-emphasized-focus);
+		color: var(--spectrum-tag-content-color-emphasized);
 	}
 }
 
 /* disabled */
 .spectrum-Tag.is-disabled {
-	border-color: var(--highcontrast-tag-border-color-disabled, var(--mod-tag-border-color-disabled, var(--spectrum-tag-border-color-disabled)));
-	background-color: var(--highcontrast-tag-background-color-disabled, var(--mod-tag-background-color-disabled, var(--spectrum-tag-background-color-disabled)));
-	color: var(--highcontrast-tag-content-color-disabled, var(--mod-tag-content-color-disabled, var(--spectrum-tag-content-color-disabled)));
+	/* mod/highcontrast colors specified at top */
+	border-color: var(--spectrum-tag-border-color-disabled);
+	background-color: var(--spectrum-tag-background-color-disabled);
+	color: var(--spectrum-tag-content-color-disabled);
 	pointer-events: none;
 
 	.spectrum-Avatar {
 		opacity: var(--mod-avatar-opacity-disabled, var(--spectrum-avatar-opacity-disabled));
 	}
+
+	.spectrum-Thumbnail {
+		opacity: var(--mod-thumbnail-opacity-disabled, var(--spectrum-thumbnail-opacity-disabled));
+	}
```

Comment:
Not in the spec but seems to work fine for now, I'll follow up and we can change it later if needed.